src/core/scene/a-scene.js

```diff
@@ -467,11 +512,29 @@ module.exports = registerElement('a-scene', {
     render: {
       value: function () {
         var effect = this.effect;
+        var camera = this.camera;
         var delta = this.clock.getDelta() * 1000;
         this.time = this.clock.elapsedTime * 1000;
+
         if (this.isPlaying) { this.tick(this.time, delta); }
+
         this.animationFrameID = effect.requestAnimationFrame(this.render);
-        effect.render(this.object3D, this.camera);
+
+        window.performance.mark('render-iteration-started');
```

Comment:
I can't readily think of any logic besides post processing that would need to happen after the scene rendering. Having tick for scene updates and tock for post proc is clean, if tock where to be generic it would lead to new issues like having to check in components whether post processing took place. Without these checks the final compositor would overwrite the screen with redundant frames